docs/en/query_language/functions/higher_order_functions.md

```diff
@@ -119,41 +119,103 @@ SELECT arrayReverseFill(x -> not isNull(x), [1, null, 3, 11, 12, null, null, 5,
 
 Note that the first argument (lambda function) can't be omitted in the `arrayReverseFill` function.
 
-### arraySplit(func, arr1, ...) {#higher_order_functions-array-split}
+### arraySplit {#array-split}
 
-Split `arr1` into multiple arrays. When `func` returns something other than 0, the array will be split on the left hand side of the element. The array will not be split before the first element.
+Creates an array of elements split into groups the length of size (return value from `func`function). When `func` returns something other than 0, the array will be split on the left hand side of the element. If array can't be split evenly, the final chunk will be the remaining elements.
 
-Examples:
+**Syntax**
 
 ```sql
-SELECT arraySplit((x, y) -> y, [1, 2, 3, 4, 5], [1, 0, 0, 1, 0]) AS res
+arraySplit(func, arr1, arr2, ..., arrN)
 ```
 
+**Parameters**
+
+- `func` — The lambda function. **Required**.
```

Comment:
* link to lambda function docs
* I think we haven't used bold to mark to mark required fields in other functions.
* Also it's not practical to mention "required" everywhere, because if something is optional you need to include a sentence about default or what happens if it's omitted. So mentioning as required could be skipped, while optional can not.

Reply:
> I think we haven't used bold to mark to mark required fields in other functions.

Will be

> link to lambda function docs

Done

>Also it's not practical to mention "required" everywhere, because if something is optional you need to include a sentence about default or what happens if it's omitted. So mentioning as required could be skipped, while optional can not.

Removed